templates/edge-computing/index.html

```diff
@@ -10,468 +13,457 @@
-                        height="612",
+            {{ image(url="https://assets.ubuntu.com/v1/a684b3b2-Hewlett-Packard-logo.png",
+                        alt="Hewlett Packard",
+                        width="141",
+                        height="156",
                         hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-logo-section__logo"}) | safe
+                        loading="auto|lazy") | safe
             }}
           </div>
           <div class="p-logo-section__item">
-            {{ image(url="https://assets.ubuntu.com/v1/8b33ba49-dell-technologies-logo-resize.png",
-                      alt="Dell",
-                      width="280",
-                      height="280",
-                      hi_def=True,
-                      loading="lazy") | safe
+            {{ image(url="https://assets.ubuntu.com/v1/8d0e5ce4-Dell-logo.png",
+                        alt="Dell",
+                        width="156",
+                        height="156",
+                        hi_def=True,
+                        loading="auto|lazy") | safe
             }}
           </div>
           <div class="p-logo-section__item">
-            {{ image(url="https://assets.ubuntu.com/v1/c7772315-hp.png",
+            {{ image(url="https://assets.ubuntu.com/v1/b466c5c3-hp-logo.png",
                         alt="HP",
-                        width="290",
-                        height="290",
+                        width="66",
+                        height="156",
                         hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-logo-section__logo"}) | safe
+                        loading="auto|lazy") | safe
             }}
           </div>
           <div class="p-logo-section__item">
-            {{ image(url="https://assets.ubuntu.com/v1/97596b4c-logo-lenovo.png",
+            {{ image(url="https://assets.ubuntu.com/v1/99418046-lenovo-logo.png",
                         alt="Lenovo",
-                        width="400",
-                        height="66",
+                        width="101",
+                        height="156",
                         hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-logo-section__logo"}) | safe
+                        loading="auto|lazy") | safe
             }}
           </div>
           <div class="p-logo-section__item">
-            {{ image(url="https://assets.ubuntu.com/v1/80ab9b3c-2018-logo-cisco.svg",
+            {{ image(url="https://assets.ubuntu.com/v1/d4dc5e10-cisco-logo.png",
                         alt="Cisco",
-                        width="146",
-                        height="146",
+                        width="144",
+                        height="156",
                         hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-logo-section__logo"}) | safe
```

Comment:
This logo looks a bit distorted.
@mattea-turic

Reply:
No, that's my bad, I applied incorrect proportions. Good catch!